src/coreclr/tools/Common/JitInterface/CorInfoBase.cs

```diff
@@ -61,7 +61,7 @@ static byte _getMethodInfo(IntPtr thisHandle, IntPtr* ppException, CORINFO_METHO
             var _this = GetThis(thisHandle);
             try
             {
-                return _this.getMethodInfo(ftn, info) ? 1 : 0;
+                return _this.getMethodInfo(ftn, info) ? (byte)1 : (byte)0;
```

Comment:
This is auto-generated file. Could you please also update the auto-generator (path to the auto-generator is at the top of the file)?